apps/user/src/app/signup/page.tsx

```diff
@@ -42,19 +44,20 @@ const SignUpPage = () => {
                             />
                             <ButtonInput
                                 label="이메일 인증"
-                                buttonText="인증"
+                                buttonText={isRequestEmail ? '재전송' : '인증'}
                                 onClick={() => {
                                     handleRequestEmailButtonClick();
```

Comment:
```suggestion
                                    if (isButtonDisabled) return;
                                    handleRequestEmailButtonClick();
```
처럼 변경해야할 것 같습니다. 이상하게 ButtonInput 컴포넌트 내에 Button 컴포넌트의 enabled 속성이 false가 되도 클릭되네요.

Reply:
한번 확인해보겠습니다~